### DISCUSSION TITLE
Change Orchid icon (the one in the sidebar) for another thing

### DISCUSSION THREAD

**Discussion by alejandroGonGon** [orchidsoftware/platform#2005] [Q&A] [answered]
Hey! I have a question.

There is a way to change the icon of Orchid and all the content in the top of the side bar menu ?

![image](https://user-images.githubusercontent.com/56725406/145853474-ad0ef18e-448f-41fe-a381-78eec2b9222d.png)

Like i want to change it to my own or customize it, idk.

**Comment by tabuna** [comment, marked as answer]
I think you should take a look at this page https://orchid.software/en/docs/brand/

**Reply by alejandroGonGon** [reply]
THANKS!
